### PR TITLE
Display team names ("applicant" for exporters).

### PR DESCRIPTION
**Pull request by alixedi**

### CODE REVIEW

File: caseworker/templates/includes/audit-trail.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
 			<div id="case-activity-{{ audit.id }}" class="app-activity__item">
 				<p class="govuk-body">
 					{# user not guaranteed to have id e.g, in case of system user #}
-					{% if audit.user.type == 'exporter' or not audit.user.id %}
-						<span class="app-activity__item__user">{{ audit.user.first_name }} {{ audit.user.last_name }}</span>
+					{% if audit.user.type == 'exporter' %}
+						<span class="app-activity__item__user">Applicant: {{ audit.user.first_name }} {{ audit.user.last_name }}</span>
```
Comment (saruniitr) [inline caseworker/templates/includes/audit-trail.html:8]: the text `Applicant:` and the user team name are required in bold as per designs.

Comment (alixedi) [reply caseworker/templates/includes/audit-trail.html:8]: They are. I think because `activity__item__user` makes them.

Comment (saruniitr) [reply caseworker/templates/includes/audit-trail.html:8]: :+1:

Comment (saruniitr) [reply caseworker/templates/includes/audit-trail.html:8]: sorry slight issue, we only want "Applicant" and "user team" in bold but not their first and last names

Comment (alixedi) [reply caseworker/templates/includes/audit-trail.html]: I've moved the `</span>` to before the user name now.

**Pull request merged**